commit 240f05c4e9c9ac258d29ce95ff2f064da3a4a772 (refs/changes/81/4981/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-10-19T14:29:38-07:00 (12 months ago)
    
    TRITON-880 want ufds to store and understand Yubikey attestation certs

diff --git a/lib/keys.js b/lib/keys.js
index ac08720..3f3414e 100644
--- a/lib/keys.js
+++ b/lib/keys.js
@@ -40,7 +40,8 @@ function translateKey(key) {
     return {
         name: key.name,
         fingerprint: key.fingerprint,
-        key: key.openssh
+        key: key.openssh,
+        attested: key.attested
     };
 }
 
@@ -75,6 +76,10 @@ function create(req, res, next) {
         name: req.params.name || null
     };
 
+    if (req.params.attestation) {
+        obj.attestation = req.params.attestation;
+    }
+
     var user, account;
     if (req.params.user) {
         user = req.params.user;
diff --git a/package.json b/package.json
index 283bab2..86cb344 100644
--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
         "semver": "5.4.1",
         "strsplit": "1.0.0",
         "triton-metrics": "0.1.0",
-        "ufds": "1.3.0",
+        "ufds": "1.4.0",
         "vasync": "2.2.0",
         "verror": "1.10.0",
         "xregexp": "3.1.0"
